packages/core/src/Modules/AccountDeactivated/account-deactivated.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Modal } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { getDerivComLink } from '@deriv/shared';
+
+class AccountDeactivated extends React.Component {
```

Comment:
(optimization)

Can this be changed to functional component to leverage hooks?

Reply:
it's connecting to the mobx store.
is there any way to use functional component with the store?